src/Symfony/Component/HttpKernel/Attribute/Serialize.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+/**
+ * Controller tag to serialize response.
+ *
+ * @author Konstantin Myakshin <molodchick@gmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_METHOD)]
+final class Serialize
```

Comment:
Hi,
I think this is a great idea and would be very useful for anyone developing webservice so thank you.
However wouldn't it be more logical for consistency to name the attribute in the the same manner as the others ?
Something like `MapResponsePayload` seems more intuitive IMHO.

Reply:
this attributes can be renamed soon, see #58709